modules/ivc_champva/app/jobs/ivc_champva/missing_form_status_job.rb

```diff
@@ -44,33 +45,56 @@ def perform # rubocop:disable Metrics/MethodLength
       Rails.logger.error e.backtrace.join("\n")
     end
 
-    def fetch_forms_by_uuid(form_uuid)
-      @fetch_forms_by_uuid ||= IvcChampvaForm.where(form_uuid:)
+    def construct_email_payload(form, template_id)
+      { email: form.email,
+        first_name: form.first_name,
+        last_name: form.last_name,
+        form_number: form.form_number,
+        file_count: nil,
+        pega_status: form.pega_status,
+        created_at: form.created_at.strftime('%B %d, %Y'),
+        template_id: template_id,
+        form_uuid: form.form_uuid }
     end
 
+    # Sends an email to user notifying them of their submission's failure
+    #
+    # @param form [IvcChampvaForm] form object in question
+    # @param template_id [string] key for template to use in `IvcChampva::Email::EMAIL_TEMPLATE_MAP`
     def send_failure_email(form, template_id)
-      form_data =
-        {
-          email: form.email,
-          first_name: form.first_name,
-          last_name: form.last_name,
-          form_number: form.form_number,
-          file_count: nil,
-          pega_status: form.pega_status,
-          created_at: form.created_at.strftime('%B %d, %Y'),
-          template_id: template_id
-        }
+      form_data = construct_email_payload(form, template_id)
       ActiveRecord::Base.transaction do
         if IvcChampva::Email.new(form_data).send_email
           fetch_forms_by_uuid(form[:form_uuid]).update_all(email_sent: true) # rubocop:disable Rails/SkipsModelValidations
         else
-          additional_context = { form_id: form[:form_number] }
+          additional_context = { form_id: form[:form_number], form_uuid: form[:form_uuid] }
           monitor.log_silent_failure(additional_context)
           raise ActiveRecord::Rollback, 'Pega Status Update/Action Required Email send failure'
         end
       end
     end
```

Comment:
Pulled the email payload construction out into its own function so it could be cleanly shared with the new `send_zsf_notification_to_pega` function.